schema/types/0_common.schema

```diff
@@ -0,0 +1,6 @@
+// This file must be the first in lexicographical order. Therefore the filename
+// is prefixed with "0_".
```

Comment:
Rather this arguably ugly hack in the filename, how about we just use an extra "base" or "common" directory in the `schema-dirs` option?

Reply:
As discussed, this is not necessary anymore and the file has been renamed to `base.schema`. Note that I also tested with the name `zbase.schema` and the loading was still successful.

Reply:
Nice!